lib/datadog/appsec/waf.rb

```diff
@@ -627,7 +627,7 @@ def run(input, timeout = LibDDWAF::DDWAF_RUN_TIMEOUT)
 
           max_container_size  = LibDDWAF::DDWAF_MAX_CONTAINER_SIZE
           max_container_depth = LibDDWAF::DDWAF_MAX_CONTAINER_DEPTH
-          max_string_length   = LibDDWAF::DDWAF_MAX_CONTAINER_SIZE
+          max_string_length   = LibDDWAF::DEFAULT_MAX_STRING_LENGTH
```

Comment:
Shouldn't this be the following?

```suggestion
          max_string_length   = LibDDWAF::DDWAF_MAX_STRING_LENGTH
```

Otherwise `DEFAULT_MAX_STRING_LENGTH` is zero.

Reply:
Absolutely. Thanks for pointing it out